<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="15.521" tests="2" failures="1">
  <testsuite name="Root Suite" timestamp="2023-06-12T20:17:33" tests="0" file="cypress\e2e\UI\trigger.js" time="0.000" failures="0">
  </testsuite>
  <testsuite name="E2E - Scrollowanie/Najechanie" timestamp="2023-06-12T20:17:33" tests="2" time="15.402" failures="1">
    <testcase name="E2E - Scrollowanie/Najechanie Najechanie na element" time="2.717" classname="Najechanie na element">
    </testcase>
    <testcase name="E2E - Scrollowanie/Najechanie Scrollowanie do boxa specials" time="4.139" classname="Scrollowanie do boxa specials">
      <failure message="Timed out retrying after 4000ms: Expected to find element: `a[title=&quot;Specials&quot;]`, but never found it." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 4000ms: Expected to find element: `a[title="Specials"]`, but never found it.
    at Context.eval (webpack:///./cypress/e2e/UI/trigger.js:12:11)]]></failure>
    </testcase>
  </testsuite>
</testsuites>